tests/neg/assignments.check

```diff
@@ -0,0 +1,22 @@
+-- [E052] Type Error: tests/neg/assignments.scala:17:8 -----------------------------------------------------------------
+17 |    x_= = 2  // error should give missing arguments, was: Reassignment to val x_=
+   |    ^^^^^^^
+   |    Bad assignment to setter should use x_=(2)
```

Comment:
Sorry, I don't see how this is an improvement. Who in their right minds would write `x_= =`? That's just not what I see, and we should not special case it.